src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,41 @@
+package racingcar.model;
+
+import racingcar.RandomManager;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private static final int MOVE_CONDITION = 3;
+
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public void moveCars() {
+        this.cars.forEach(car -> car.move(isMovingForward(RandomManager.getInstance().getRandomValue())));
+    }
+
+    public static boolean isMovingForward(int value) {
+        return value > MOVE_CONDITION;
+    }
+
+    public List<Car> getWinnerCars() {
+        int maxDistance = getMaxDistance();
+        return this.cars.stream()
+                .filter(car -> car.isMaxDistance(maxDistance))
+                .collect(Collectors.toList());
+    }
+
+    private int getMaxDistance() {
+        return this.cars.stream().map(Car::getDistance).max(Integer::compareTo).orElseThrow(NoSuchElementException::new);
```

Comment:
가로로 늘어지면 아주 조금 읽기 불편한 것 같아요
모든 사람이 코드를 볼 때 큰 화면에서 본다는 보장도 없구요
그래서 어느정도 구분을 짓고 개행을 넣어주면 좋겠습니다.
```suggestion
    private int getMaxDistance() {
        return this.cars.stream().map(Car::getDistance)
        .max(Integer::compareTo)
        .orElseThrow(() -> NoSuchElementException("메시지"));
```

예외에는 항상 메시지를 넣어주는 것이 좋습니다. 😄